### DISCUSSION TITLE
Obstacle One

### DISCUSSION THREAD

**Discussion by DebravdH** [stellar/sorobanathon#50] [General]
Our artist rendition of Story 2 was a group effort. Enjoy! -Debra, Erik and Rylee
https://litemint.com/items/GD7MZWPW5VLVOUR3GE2T24LNIUAMAQGRKSDSA6VI5KPNVJRB3MCORDVO/ObstacleOne

**Comment by stellarjohn** [comment]
Howdy DebravdH,

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 300 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.
Please let us know if you have any questions.

Cheers!
